docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -627,9 +596,78 @@ simplifying assumptions that we have included:
 can use an interactive simulation in a Jupyter Notebook to check that
 simulants are moving at the expected rates.
 
+.. _census_prl_international_immigration:
 
-2.3.5 Component 9: Address
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.3.5 Component 9: International Immigration (In-Migration)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New simulants are added by migration into the US from other countries.
+We simulate two kinds of immigration: household moves and non-reference-person moves.
+
+#. A **household move** is when an entire household (which may be a single person) enters from outside the country as a unit,
+   preserving relationships within the unit.
+#. A **non-reference-person move** is when a person enters from outside the country and joins an existing household.
+   Non-reference-person moves are independent, single-person events that do not preserve relationship structure.
+
+For the purposes of immigration, a group quarters person should be treated identically to a one-person household,
+with the group quarters person as the reference person.
+This means that group quarters people should only enter the US in a household move.
```

Comment:
could you clarify this a bit? are you referring to group quarters individuals or "households" moving into the US?
if individuals, is this group quarters individuals moving into group quarters, into a non-group quarters existing household, or just into their own household (then they become household head?)
if it's an entire group quarters "household", i assume this would mean they move into the correct group quarters type?

Reply:
GQ people move as a "household" of one person. They would move to the correct (sampled) group quarters type and address. They would never join an existing household.

I will attempt to reword this to make it more clear.

Reply:
Actually, I have a question for you: is there any correlation between group quarters type and address? Does each simulant in a group quarters type (e.g. carceral) have a unique address?